src/stories/containers/Endgame/EndgameContainer.tsx

```diff
@@ -7,7 +7,7 @@ import { toAbsoluteURL } from '@ses/core/utils/urls';
 import React from 'react';
 import BudgetStructureSection from './components/BudgetStructureSection/BudgetStructureSection';
 import BudgetTransitionStatusSection from './components/BudgetTransitionStatusSection/BudgetTransitionStatusSection';
-import EndgameIntroductionBanner from './components/EndgameIntroductionBanner/EndgameIntroductionBanner';
+import EndgameIntroductionBannerReskin from './components/EndgameIntroductionBannerReskin/EndgameIntroductionBannerReskin';
```

Comment:
Instead of using the word `Reskin`, use `legacy` in the old component, as this component is the final implementation

Reply:
you are right, done